app/views/docs/oauth-providers/linkedin.md

```diff
@@ -0,0 +1,358 @@
+# LinkedIn provider
+
+Appwrite allows you to authenticate users using their LinkedIn account through the LinkedIn OAuth2 provider. OAuth authentication is a great way to reduce friction for your users and increase user conversion by simplifying the signup process. You can learn more about Appwrite's other OAuth2 providers [here](placeholder link).
+
+## Enabling the LinkedIn provider
+
+Before you can use LinkedIn to authenticate users, you need to enable the provider in your Appwrite console.
+
+1. Navigate to your Appwrite project
+2. Navigate to **Auth** > **Settings**
+3. Find and open the OAuth provider
+4. In the **Linkedin OAuth2 Settings** modal, use the toggle to enable the provider
+
+Don't close this modal, we'll need to create a LinkedIn OAuth app to complete this form.
+
+## Creating a LinkedIn Developer app
+
+To use LinkedIn OAuth with Appwrite, you need to create a LinkedIn Developer app. You can do this by following the [How to Implement OAuth](https://learn.microsoft.com/en-us/linkedin/shared/authentication/authorization-code-flow?tabs=HTTPS1#how-to-implement-3-legged-oauth) guide from LinkedIn.
```

Comment:
@Dksie09 As per DM, find a better post?

Reply:
Steven confirmed it's 3 legged, so should we just keep it this way?